tests/test_crs/valid_fiod_store.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+  namespace: ripsaw
+spec:
+  cleanup: false
+  elasticsearch:
```

Comment:
Should we also create a CR that stores results but does not index?

Reply:
I like that idea, but i'd say let's merge this and we'll come back to this later and add it. As there needs to be a wider discussion that needs to occur i.e. not just for fio-d but if it needs to be a norm or not.